### DISCUSSION TITLE
How to install the Zerox86 emulator?

### DISCUSSION THREAD

**Discussion by Santorayo3** [eduardofilo/RG350_adam_image#139] [General]
The manual states that there is an alternative DOS Emulator.
So I've downloaded the .opk file and put it in my apps folder. But it doesn't show up anywhere in SimpleMenu.

I've found the emulator's homepage http://zerox86.patrickaalto.com/zfaq.html and followed the steps to configure the path but I just don't know how to start the emulator from the Adam Image. 😅

**Comment by eduardofilo** [comment]
SimpleMenu only shows the OPKs that belong to the applications and games categories in the apps & games section. You would have to build a system for that emulator to appear in SimpleMenu or edit the OPK so that it belongs to one of those categories. For now you can open that emulator from GMenu2X. Disable SimpleMenu as the default launcher and you will find zerox86 in the emulators section.

**Comment by Santorayo3** [comment]
Thanks. I got it to run but it seem incompatible with the RG280V... besides the D-Pad none of the buttons work. :/
